automation/project/Dependencies.scala

```diff
@@ -7,11 +7,11 @@ object Dependencies {
   val akkaHttpV     = "10.2.0"
   val jacksonV      = "2.17.1"
 
-  val workbenchLibsHash = "3a8f98c"
-  val serviceTestV = s"4.4-${workbenchLibsHash}"
-  val workbenchGoogleV = s"0.30-${workbenchLibsHash}"
+  val workbenchLibsHash = "9138393"
```

Comment:
Could you also update the other one in automation? https://github.com/broadinstitute/rawls/blob/21ca8938dc60328c2f927089c8ed45be46820dd6/automation/project/Dependencies.scala#L10

Reply:
@cahrens I'm confused - that _is_ the line I updated?

Reply:
I just pushed a commit to update the version in runtime code too, if that addresses your comment

Reply:
Ha! Yes, I just meant "the other one" and was assuming you hadn't changed automation. Thank you!